File: abc295/abc295_a.py

def main():

    WORDS = ["and", "not", "that", "the", "you"]

    N = int(input())
    W = list(input().split(' '))

    for i in range(N):
        if W[i] in WORDS:
            print("Yes")
            exit()

    print("No")


main()

File: abc295/abc295_a.cpp

#include <bits/stdc++.h>
using namespace std;

int main()
{
	int N;
	cin >> N;

	const set<string> words = {"and", "not", "that", "the", "you"};

	for (int i = 0; i < N; i++)
	{
		string W;
		cin >> W;

		if (words.count(W))
		{
			cout << "Yes" << endl;
			exit(0);
		}
	}

	cout << "No" << endl;

	return 0;
}